public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,28 @@
+importScripts("https://www.gstatic.com/firebasejs/5.9.4/firebase-app.js");
+importScripts("https://www.gstatic.com/firebasejs/5.9.4/firebase-messaging.js");
+
+firebase.initializeApp({
+  messagingSenderId: "220731667069"
```

Comment:
Attention

Reply:
Yeah, have to pull from process.env

Reply:
So the things is that, we can't use the process.env in this file and manifest.json so I'll just add step in readme.md